client/my-sites/scan/navigation.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SectionNav from 'components/section-nav';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import { getSelectedSite } from 'state/ui/selectors';
+
+interface Props {
+	section: string;
+}
+
+export default function ScanNavigation( { section }: Props ) {
+	const site = useSelector( getSelectedSite );
+	const translate = useTranslate();
+
+	if ( ! site ) {
+		return <SectionNav />;
+	}
+
+	const strings = {
+		scanner: translate( 'Scanner', { context: 'scan screen' } ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Scanner' )` *ES Score: 15*


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.